src/libostree/ostree-sepolicy.c

```diff
@@ -257,6 +260,18 @@ initable_init (GInitable     *initable,
         freecon (con);
       }
 
+      {
+        g_autofree guchar *csum_raw = NULL;
+        if (!ostree_checksum_file (etc_selinux_dir, OSTREE_OBJECT_TYPE_FILE,
```

Comment:
Also I'm a bit confused...we're checksumming `etc_selinux_dir`, the directory??

Also regarding policy and checksums, see http://marc.info/?l=selinux&m=145677940526857&w=2